MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1693,7 +1701,7 @@
         _properties.limits.maxTessellationControlPerVertexInputComponents = _properties.limits.maxVertexOutputComponents;
         _properties.limits.maxTessellationControlPerVertexOutputComponents = _properties.limits.maxTessellationControlPerVertexInputComponents;
         // Reserve a few for the tessellation levels.
-        _properties.limits.maxTessellationControlPerPatchOutputComponents = _properties.limits.maxFragmentInputComponents - 8;
+        _properties.limits.maxTessellationControlPerPatchOutputComponents = std::max(_properties.limits.maxFragmentInputComponents - 8, 120u);
```

Comment:
@cdavis5e 

Since the current max for `maxFragmentInputComponents` is 124, looks like this is always going to be 120.

Are you just future-proofing this in case future OS's bump `maxFragmentInputComponents` higher?

Reply:
> Are you just future-proofing this in case future OS's bump `maxFragmentInputComponents` higher?

Yes.